src/_data/config.json

```diff
@@ -10,6 +10,11 @@
       "code": "fr-CA",
       "dir": "ltr",
       "name": "Français"
+    },
+    {
+      "code": "fr-FR",
+      "dir": "ltr",
+      "name": "Français"
```

Comment:
The double Français in the list of languages is a bit confusing to look at. I'm guessing you can't actually require much here, but it might be better to have as an example `Français (fr-FR)` and `Français (fr-CA)`. Or some other way to easily distinguish them.

<img width="836" alt="Screen Shot 2021-07-19 at 10 03 34 AM" src="https://user-images.githubusercontent.com/556024/126177524-5934c481-0b25-42c9-9658-ed8c46e1bea4.png">